src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/LicenceIntegrationTest.kt

```diff
@@ -451,6 +482,12 @@ class LicenceIntegrationTest : IntegrationTestBase() {
       conditions = listOf("Condition 1", "Condition 2", "Condition 3")
     )
 
+    val anAdditionalConditionsRequest = AdditionalConditionsRequest(
+      additionalConditions = listOf(
+        AdditionalCondition(code = "code", sequence = 0, text = "text")
+      )
```

Comment:
Might be worth trying this with several additional conditions to ensure iterating over a number of them?